packages/wallet/primitives/src/network.ts

```diff
@@ -1,630 +1,1229 @@
-export type Network = {
+export enum NetworkType {
+  MAINNET = 'mainnet',
+  TESTNET = 'testnet',
+  LOCAL = 'local',
```

Comment:
We don't need this LOCAL network type. We should remove the HARDHAT networks from this list as they are for testing only and don't want to be exposed